src/app/work-item/work-item-list/work-item-detail/work-item-link/work-item-link.component.scss

```diff
@@ -2,193 +2,199 @@
-            .weight-800{ font-weight: 800; }
+        margin: 0;
+        padding: em(5) 0;
+        font-weight: 800;
+      }
+      .link-list-type-header {
+        font-weight: 600;
+      }
+      .link-list{
+        display: flex;
+        justify-content: space-between;
+        align-items: center;
+        padding: em(2) em(5);
+        cursor: pointer;
+        &:hover{
+          background-color: $color-pf-black-300;
         }
-        .search-result {
-            position: absolute;
-            top: 0;
-            width: $width100;
-            z-index: 10;
-            box-shadow: 0 1px 10px 0 rgba(192, 192, 192, 0.7);
-            max-height: em(200);
-            overflow-y: auto;
-            li{
-                background-color: $color-pf-white;
-                padding: em(5);
-                border-bottom: 1px solid $color-pf-black-300;
-                border-left: 1px solid $color-pf-black-300;
-                border-right: 1px solid $color-pf-black-300;
-                &:hover {
-                    background-color: $color-pf-black-500;
-                }
-                &.selected {
-                    background-color: $color-pf-black-200;
-                }
-            }
+        .link-title {
+          width: em(150);
+          max-width: em(150);
+          text-overflow: ellipsis;
+          overflow: hidden;
+          white-space: nowrap;
         }
+      }
+    }
+    .workitem-link-search-box {
+      width: $width100;
+      padding: em(2) em(5);
+      outline: none;
     }
+    .search-result-container {
+      position: relative;
+      .weight-800{ font-weight: 800; }
+    }
+    .search-result {
+      position: absolute;
+      top: 0;
+      width: $width100;
+      z-index: 10;
+      box-shadow: 0 1px 10px 0 rgba(192, 192, 192, 0.7);
+      max-height: em(200);
+      overflow-y: auto;
+        li{
+          background-color: $color-pf-white;
+          padding: em(5);
+          border-bottom: 1px solid $color-pf-black-300;
+          border-left: 1px solid $color-pf-black-300;
+          border-right: 1px solid $color-pf-black-300;
+          &:hover {
+            background-color: $color-pf-black-500;
+          }
+          &.selected {
+            background-color: $color-pf-black-200;
+          }
+        }
+      }
+  }
 }
 // .link-header > div{
 //     margin: 15px;
 // }
 @media (max-width: $screen-md-min){
-    .link-container {
-        .link-header{
-            .content__title{
-                padding-left: em(5);
-            }
-            .link-header-nav{
-                 padding: em(5);
-                span{
-                    float: none;
-                    display: block;
-                    line-height: 1.2;
-                }
-                .link-header-nav-text{
-                    padding: 0;
-                }
-            }
+  .link-container {
+    .link-header{
+      .link-header-nav-list{
+        .link-header-nav{
+          span{
```

Comment:
Discussed: this is to keep the specificity in check with their initial value.
This can be cleaned up later.